subworkflows/local/methylation/tests/nextflow.config

```diff
@@ -0,0 +1,130 @@
+params {
+    preset = "pacbio"
+}
+
+process {
+    withName: 'PREPARE_GENOME:UNTAR_VEP_CACHE' {
+        ext.prefix = { "${name}".contains('merged') ? './vep_cache/homo_sapiens_merged/' : './vep_cache/homo_sapiens/' }
+    }
+    withName: 'SHORT_VARIANT_CALLING:DEEPVARIANT' {
+        ext.prefix = { intervals ? "${meta.id}_${intervals}_deepvariant" : "${meta.id}_deepvariant" }
+        ext.args = { [
+            '--model_type WGS',
+            "--sample_name=${meta.id}",
+            '-vcf_stats_report=False'
+        ].join(' ') }
+    }
+
+    withName: 'SHORT_VARIANT_CALLING:GLNEXUS' {
+        ext.args = '--config DeepVariant_unfiltered'
+    }
+
+    withName: 'SHORT_VARIANT_CALLING:BCFTOOLS_CONCAT' {
+        ext.prefix = { "${meta.id}_concat" }
+        ext.args = [
+            '--no-version',
+            '--allow-overlaps'
+        ].join(' ')
+    }
+
+    withName: 'SHORT_VARIANT_CALLING:BCFTOOLS_NORM_MULTISAMPLE' {
+        ext.prefix = { "${meta.id}_norm_multisample" }
+        ext.args = [
+            '--no-version',
+            '-m -',
+            '--output-type z',
+            '--write-index=tbi',
+            '-w 10000'
+        ].join(' ')
+    }
+
+    withName: 'SHORT_VARIANT_CALLING:BCFTOOLS_NORM_SINGLESAMPLE' {
+        ext.prefix = { "${meta.id}_norm_singlesample" }
+        ext.args = [
+            '--no-version',
+            '-m -',
+            '-w 10000',
+            '--output-type z',
+            '--write-index=tbi',
+        ].join(' ')
+    }
+
+    withName: 'SHORT_VARIANT_CALLING:BCFTOOLS_FILLTAGS' {
+        ext.prefix = { "${meta.id}_ac" }
+        ext.args = [
+            '--no-version',
+            '--output-type u'
+        ].join(' ')
+    }
+
+    withName: 'MINIMAP2_ALIGN' {
+        ext.args = '-x map-hifi'
+    }
```

Comment:
Have you checked what the output files from modkit look like? 

Without `-y` here no methylation tags should be copied over during alignment, and no (meaningful) pileups should be able to be performed?